src/extensions/default/JavaScriptRefactoring/RenameIdentifier.js

```diff
@@ -88,7 +88,8 @@ define(function (require, exports, module) {
         var result = new $.Deferred();
 
         function isInSameFile(obj, refsResp) {
-            return (obj && obj.file === refsResp.file);
+            // In case of unsaved files, After renameing once Tern is returning filename without forward slash
+            return (obj && (obj.file === refsResp.file || (obj.file === refsResp.file.slice(1, refsResp.file.length))));
```

Comment:
Can be written as -
`return (obj && (obj.file === refsResp.file || obj.file === refsResp.file.slice(1, refsResp.file.length)));`

By eliminating the last brace.

Reply:
@swmitra Done. Please merge this PR.